tests/jlm/llvm/frontend/llvm/LoadTests.cpp

```diff
@@ -25,7 +25,8 @@ LoadConversion()
   std::unique_ptr<Module> llvmModule(new Module("module", context));
 
   auto int64Type = Type::getInt64Ty(context);
-  auto pointerType = Type::getInt64PtrTy(context);
+  // ADDRESS_SPACE_GENERIC == 0
+  auto pointerType = llvm::PointerType::get(Type::getInt64Ty(context), 0);
```

Comment:
This was what I managed to find as a replacement that works for the test.
The strange part is that the AddressSpace enum seems to be related to Nvidia PTX, which is the reason that I pass in '0'.

Reply:
I think the `PointerType::getUnqual()` is the right one to use here: https://llvm.org/doxygen/classllvm_1_1PointerType.html